BusBoard.Web/ViewModels/BusInfo.cs

```diff
@@ -1,13 +1,51 @@
-﻿namespace BusBoard.Web.ViewModels
+﻿using System.Collections.Generic;
+using System.Linq;
+using BusBoard.Api;
+
+namespace BusBoard.Web.ViewModels
 {
-  public class BusInfo
-  {
-    public BusInfo(string postCode)
+    public class BusInfo
     {
-      PostCode = postCode;
-    }
+        public string ConfirmationMessage { get; set; }
+        public List<StopInfo> StopInfos { get; }
+
+        public BusInfo(string postcode)
+        {
+            StopInfos = new List<StopInfo>();
```

Comment:
Do you need this? You could just set it to empty list when you declare the variable 🙂